ElectronicObserver/Utility/Data/EquipmentFitBonus.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System.Collections.Generic;
+using ElectronicObserverTypes;
+using System.Linq;
+using ElectronicObserverTypes.Serialization.FitBonus;
+
+namespace ElectronicObserver.Utility.Data;
+
+public static class EquipmentFitBonus
+{
+
+	public static FitBonusValue GetFitBonus(this IShipData ship, IList<FitBonusPerEquipment> bonusList) => 
+		GetFitBonuses(ship, bonusList)
+		.Aggregate(
+			new FitBonusValue(),
+			(bonusA, bonusB) => bonusA + bonusB,
+			bonus => bonus
+		);
+
+	/// <summary>
+	/// Keep in mind that accuracy bonus is included
+	/// </summary>
+	/// <param name="ship"></param>
+	/// <returns></returns>
+	public static List<FitBonusValue> GetFitBonuses(this IShipData ship, IList<FitBonusPerEquipment> bonusList)
+	{
+		IList<IEquipmentData> equipments = ship.AllSlotInstance.Where(equipment => equipment != null).ToList()!;
+		if (ship.ExpansionSlotInstance != null) equipments.Add(ship.ExpansionSlotInstance);
+
+		return GetFitBonuses(equipments, ship.MasterShip, bonusList);
+	}
+
+	/// <summary>
+	/// Keep in mind that accuracy bonus is included
+	/// </summary>
+	/// <param name="equipments"></param>
+	/// <param name="shipMaster"></param>
+	/// <param name="bonusList"></param>
+	/// <returns></returns>
+	public static List<FitBonusValue> GetFitBonuses(IList<IEquipmentData> equipments, IShipDataMaster shipMaster, IList<FitBonusPerEquipment> bonusList)
+	{
+		bool hasSurfaceRadar = equipments
+			.Any(equipment => equipment.MasterEquipment.IsSurfaceRadar);
+
+		bool hasAirRadar = equipments
```

Comment:
We have an extension for [HasAirRadar](https://github.com/gre4bee/ElectronicObserver/blob/main/ElectronicObserverTypes/Extensions/ShipDataExtensions.cs#L83).